modules/core/src/matmul.dispatch.cpp

```diff
@@ -59,6 +60,138 @@ namespace cv
 *                                         GEMM                                           *
 \****************************************************************************************/
 
+#ifdef HAVE_CLBLAST
+
+// matD = alpha * matA * matB + beta * matC
+static bool ocl_gemm_clblast(InputArray matA, InputArray matB, double alpha,
+                             InputArray matC, double beta, OutputArray matD, int flags) {
+    int type = matA.type(), depth = CV_MAT_DEPTH(type), esz = CV_ELEM_SIZE(type);
+    bool haveC = matC.kind() != cv::_InputArray::NONE;
+    CV_CheckEQ(matB.type(), type, "Type of matB does not match the type of matA");
+    if (haveC) {
+        CV_CheckEQ(matC.type(), type, "Type of matC does not match the type of matA");
+    }
+    auto &device = ocl::Device::getDefault();
+    bool SupportFP64 = device.hasFP64(),
```

Comment:
variables should start from lowercase letter.

Reply:
Done